examples/developer-tools/src/controller.ts

```diff
@@ -295,18 +308,25 @@ export class Controller {
 
     if (!e.dataTransfer.items) return;
 
-    const firstItem = [...e.dataTransfer.items].find((item) => {
-      // Get the first plain text file, in case the user uploaded multiple
+    const allResults: UploadResults = new Map();
+    const allPromises: Array<Promise<void>> = [];
+
+    [...e.dataTransfer.items].forEach((item) => {
```

Comment:
Is `e.dataTransfer.items` not already iterable?
```suggestion
    e.dataTransfer.items.forEach((item) => {
```
(Though in fact I'd use a `for … of` loop rather than `.forEach`, since that obviates the need for the arrow function.  Alas GitHub won't let me suggest that change here.)